### ISSUE TITLE
Priority equivocates about what it is for (web pages)

### ISSUE THREAD

**Issue by martinthomson** [httpwg/http-extensions#1713]
The introductory material tries to be general, but routinely fails to be, falling back on talking about rendering web pages.  If this is the best example, it could be more direct about that.  Or, if this is general, it could try to be more careful about limiting its use of "document" and so forth to examples.

For example, this lacks sufficient context:

> For example, a server, with knowledge of the document structure, might want to prioritize the delivery of images 

And this launches into talking about HTML without saying HTML:

> To provide meaningful presentation of a document at the earliest moment, it is important for an HTTP server to prioritize the HTTP responses, or the chunks of those HTTP responses, that it sends.

**Comment by LPardue** [comment]
good catch. I do think this is the best example and therefore will take your first suggestion. I'll follow up with a PR.
